client/src/assets/blog/select_water_filter.ts

```diff
@@ -0,0 +1,139 @@
+import { BlogPost } from '@/assets/blog/blog_post';
+
+export const selectWaterFilter: BlogPost = {
+  title: 'How to Select a Water Filter',
+  image: 'next-steps.jpg',
```

Comment:
Could you add a different image bc we don't want duplicates of this photo for the blog home page

Reply:
done